packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -3176,7 +3226,9 @@ describe('scope hoisting', function() {
       });
     });
 
-    it("doesn't ignore missing import specifiers in source assets", async function() {
+    it.skip("doesn't ignore missing import specifiers in source assets", async function() {
+      // Skipping. Tradeoff between extra potentially unused symbols that could be tree-shaken vs error message on unused exports.
+      // Possibly only do this in development. Or just use a linter for this.
```

Comment:
Should we change the test then to make sure they are tree shaken and there's no build error?

Reply:
We could. Mainly I left the test so we could decide what we wanted to do about it. Do you wanna update it? 😉